connect_ext_ppr/webapp.py

```diff
@@ -37,6 +39,37 @@
 )
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
 
+    @router.get(
+        '/deployments/{deployment_id}',
+        summary='Deployment details',
+        response_model=DeploymentSchema,
+    )
+    def get_deployment(
+        self,
+        deployment_id: str,
+        client: ConnectClient = Depends(get_installation_client),
+        db: VerboseBaseSession = Depends(get_db),
+        installation: dict = Depends(get_installation),
+    ):
+        dep = (
+            db.query(Deployment)
+            .filter_by(id=deployment_id, account_id=installation['owner']['id'])
+            .first()
+        )
+        if dep is None:
+            raise ObjectNotFound(deployment_id)
+
+        listings = get_all_info(client)
```

Comment:
It does not make sense to collect all listing, marketplace and hub data for the account and only use a very specific part of it.
We can just get data using Connect API for specific product and hub that is associated with the deployment object.

Reply:
Ok, about [function name](https://github.com/cloudblue/connect-extension-xvs/pull/3#discussion_r1253264370), will change that in another PR. And the information that we collect, I agree with you that in case of list deployments we can change this behaviour, but for first time creation of Deployment objects we need all listing "listed", so we can leave it as it is for now.

Reply:
I changed the way we get information from Connect related to the only deployment.
But now when we use Connect `GET /products/{product_id}` endpoint, we receive the full product's data, and there are a lot of redundant information. Maybe there is a better solution if we need only object's reference?